packages/mdc-tab-scroller/README.md

```diff
@@ -111,3 +115,4 @@ Method Signature | Description
 `scrollTo(scrollX: number) => void` | Scrolls to the `scrollX` value.
 `incrementScroll(scrollX: number) => void` | Increments the current scroll value by the `scrollX` value.
 `getScrollPosition() => number` | Returns the current visual scroll position.
+`getScrollContentWidth() => number` | Returns the width of the scroll content element.
```

Comment:
Is this actually needed on the foundation, or could we simply write a one-liner referencing the correct node in the component?